packages/datadog-instrumentations/src/jest.js

```diff
@@ -430,7 +427,7 @@ addHook({
     const { tests } = testPaths
 
     const filteredTests = tests.filter(({ path: testPath }) => {
-      const relativePath = testPath.replace(`${rootDir}/`, '')
+      const relativePath = getTestSuitePath(testPath, rootDir)
```

Comment:
Any way to test this?

Reply:
it'd either require a full e2e setup, with a mocked backend, or the addition of a unit test just for this function. Given that it was buggy, the second option might make sense. Let me think about it

Reply:
ended up adding a unit test